dotCMS/src/main/java/com/dotcms/graphql/datafetcher/FileFieldDataFetcher.java

```diff
@@ -1,20 +1,23 @@
 package com.dotcms.graphql.datafetcher;
 
+import com.dotcms.graphql.DotGraphQLContext;
 import com.dotmarketing.business.APILocator;
 import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
 
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 
 public class FileFieldDataFetcher implements DataFetcher<Contentlet> {
     @Override
     public Contentlet get(final DataFetchingEnvironment environment) throws Exception {
+        final User user = ((DotGraphQLContext) environment.getContext()).getUser();
```

Comment:
A bit more readable to use
```
DotGraphQLContext.class.cast(environment.getContext()).getUser();

Reply:
did and IntelliJ is telling to reverse it like I had it before :D